src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const status = person.isMarried
+    ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+    : 'I am not married';
+
+  return (
+    <>
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age && <p className="Person__age">I am {person.age}</p>}
+      <p className="Person__partner">{status}</p>
+    </>
+  );
+};
+
+export default Person;
```

Comment:
The `Person` component is exported both as a named export on line 1 and as a default export on line 15. This redundancy can cause confusion. Consider removing the default export on line 15 if you intend to use named exports, or vice versa.